controller/templates/confd_settings.py

```diff
@@ -79,3 +79,13 @@
 PRIVATE_EXT_REGISTRY_FLAVOR = '{{ getv "/deis/controller/private_ext_registry/flavor" }}'
 PRIVATE_EXT_REGISTRY_DATA = json.loads('''{{ getv "/deis/controller/private_ext_registry/data" }}'''.replace("\'", '"'))
 {{ end }}
+
+# if defined, then scheduler will distribute the units cross that list.
+# it is a list of anything, the scheduler would pick az = AVAILABILITY_ZONE_LIST[num % len(AVAILABILITY_ZONE_LIST)]
+# which means if list is ['a', 'b'], and it is the 3rd container of the same application,
+# we would do, 3 % 2 = 1 => az = list(1) => az=b
+{{ if exists "/deis/controller/AZs" }}
```

Comment:
It is not exactly the same thing. I update 3 value is private_ext_registry exists. 
If I do it in one line, I would need to repeat 
```{{ if exists "/deis/controller/private_ext_registry/flavor" }}``` 
for every value, which we don't want because we want it to fail flavor is set and data is not.